src/services/auth/signIn.js

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+
+export const signIn = async ({
+	loginData,
+	userData,
+	dispatch,
+	initialFormState,
+	toggleAuth,
+	rememberMe,
+	toggleLoader,
+}) => {
+	try {
+		toggleLoader();
+
+		const res = await axios.post("/api/auth/login", loginData);
+		if (res.status === 200) {
+			localStorage.setItem("storeToken", res.data.encodedToken);
+			toggleAuth();
```

Comment:
The token should not be stored in localstorage for a reason, because localstorage won't get cleared automatically, and for basic xss attackers/hackers, it's easy to access those stored tokens from localstorage and modify the user data by sending any request, after all, a token is everything on e API.
To store tokens on the client-side, one way is to store them in httpOnly cookies which can be set to get cleared some time. 
For better understanding you can search it on the web regarding "where should we store the API token storage"

Reply:
Ok. will look into it. Thanks for detailed review.